src/main/kotlin/com/healthmetrix/labres/GlobalErrorHandler.kt

```diff
@@ -13,17 +17,39 @@ class GlobalErrorHandler {
 
     @ExceptionHandler(Exception::class)
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
-    fun onException(exception: Exception): ResponseEntity<InternalServerError> {
+    fun onException(exception: Exception) = handle(exception)
+
+    @ExceptionHandler(HttpRequestMethodNotSupportedException::class)
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
```

Comment:
Non of these should be `HttpStatus.INTERNAL_SERVER_ERROR` imo. Should be bad request

Reply:
Didn't want to change the external API in this convenience PR

Reply:
I will create a ticket for it and approve then.